assets/css/settings.scss

```diff
@@ -535,3 +535,114 @@
 		}
 	}
 }
+
+@font-face {
+	font-family: 'Reforma 1918-Blanca';
+	src: url( '../css/fonts/Reforma1918-Blanca.otf' );
+	font-weight: normal;
+	font-style: normal;
+}
+
+@media screen and (max-width: 780px) {
```

Comment:
It would be preferable to use a mobile-first approach here (i.e. always use `min-width` media queries to build on top of the mobile design) and not mix and match `min-width` and `max-width`, which can be a bit confusing.

Reply:
Changed: 
1. [banner](https://github.com/Automattic/sensei/pull/4801/files#diff-5c437b51ca7067e0a3307a080cfa9c2315d7fe7d62744683bd4972bbd62869deR547)

2. [background-image](https://github.com/Automattic/sensei/pull/4801/files#diff-5c437b51ca7067e0a3307a080cfa9c2315d7fe7d62744683bd4972bbd62869deR616)